subsys/bluetooth/host/settings.c

```diff
@@ -121,6 +121,18 @@ static int set(int argc, char **argv, char *val)
 		return 0;
 	}
 
+	if (!strcmp(argv[0], "name")) {
+		len = sizeof(bt_dev.name);
```

Comment:
Usage of "len" seems unnecessary ere since sizeof(bt_dev.name) is a compile time constant (i.e. you can just use that everywhere you use len at the moment)

Reply:
Done.